main/glossary/README.md

```diff
@@ -67,151 +104,379 @@ synchronously compared for structural equivalence.
+the user either gets what they said they wanted, or gets back (gets a refund) what they originally offered and escrowed.
+
 ## Payment
-Payments hold [amounts](#amounts) of certain assets 
-issued by [Mints](#mint). Specifically amounts that are *in transit* from one party to another. 
-Amounts from payments can be deposited in [purses](#purse), but otherwise, the entire amount is 
-available when the payment is transferred. Payments can be converted to [Purses](#purse). All contents 
-of a purse must be of the same [brand](#brand).
-
-See the [ERTP Guide's Payments section](/ertp/guide/purses-and-payments.md#payments)
-and the [ERTP API's Payments section](/ertp/api/payment.md#payment-getallegedbrand) 
-for more information.
-
-## Presence
-A local version of a remote object that serves as a proxy for the remote object. 
+
+Payments hold assets issued by [Mints](#mint). Specifically assets intended for transfer 
+from one party to another. All assets of a payment must be of the same [brand](#brand).
+
+For more information, see the [ERTP Guide's Payments section](/ertp/guide/purses-and-payments.md#purses-and-payments)
+and the [ERTP API's Payments section](/ertp/api/payment.md).
+
+## Payout
+The assets paid out to a user when an [seat](#seat) exits, either successfully or not. The payout is always
+what the seat's current [allocation](#allocation) is. 
+
+If there was a previous reallocation, the payout is different than what the user escrowed. If there is no reallocation
+before the seat exits, the payout is the same as what they escrowed.
+
+## Petname
+
+Petnames are your personal names for objects. No one else can see or modify a petname without your permission. 
+Think of them as similar to a phone's contacts list. The actual phone number is what a phone uses to call 
+someone, but to more easily tell who a number is associated with, it's assigned a petname, such 
+as Mom, Grandpa, Kate S., etc. In the Agoric platform, petnames are used in [wallets](#wallet).
+
+## Presence 
+A local version of a remote object that serves as the remote object's proxy. 
 If `obj` is a presence of a remote object, you can send messages to the remote object by using `E()` on `obj`. 
-See the [JavaScript Distributed Programming Guide](/distributed-programming.md) for more information. 
+For more information, see the [JavaScript Distributed Programming Guide](/distributed-programming.md). 
+
+## Proposal
+
+Proposals are records with `give`, `want`, and `exit` keys. [Offers](#offer) must include a proposal, which states
+what asset you want, what asset you will give for it, and how/when the offer maker can cancel the offer
+(see [Exit Rule](#exit-rule) for details on the last). For example:
+```
+const myProposal = harden({
+  give: { Asset: quatloosAmountMath.make(4)},
+  want: { Price: moolaAmountMath.make(15) },
+  exit: { 'onDemand'
+})
```

Comment:
```suggestion
const myProposal = harden({
  give: { Asset: quatloosAmountMath.make(4)},
  want: { Price: moolaAmountMath.make(15) },
  exit: { 'onDemand' },
})
```

This code wouldn't run due to the missing `}`. Can we move this into a tested code file and import it to prevent things like this?